### PR TITLE
Viewer About popup

### PR DESCRIPTION
**Pull request by ubitux**
Submitted as a draft because content might have to be refined.

### CODE REVIEW

File: pynopegl-utils/pynopegl_utils/viewer/__init__.py [hunk 1]
```diff
@@ -39,6 +40,35 @@
...
+
+    This software also relies on [nope.media]({_NOPE_MEDIA}), which is under
+    LGPL v2.1.
+
+    Finally, it depends on the following third-party libraries:
+    - [FFmpeg](https://www.ffmpeg.org): LGPL v2.1 / GPL v3
+    - [Harfbuzz](https://harfbuzz.github.io): MIT
+    - [FriBidi](https://github.com/fribidi/fribidi): LGPL v2.1
+    - [FreeType](https://freetype.org): FTL / GPL v2
+    - [Qt6](https://www.qt.io): GPL v3
```
Comment (mbouron) [inline pynopegl-utils/pynopegl_utils/viewer/__init__.py:68]: We should add MoltenVK on macOS (even though the viewer does not use it yet, nopegl is linked against it).

Comment (ubitux) [reply pynopegl-utils/pynopegl_utils/viewer/__init__.py:68]: Added

File: pynopegl-utils/pynopegl_utils/viewer/__init__.py [hunk 2]
```diff
@@ -39,6 +40,39 @@
...
+    f"""\
+    # About
+
+    The Nope viewer is part of the [Nope Foundry]({_NOPE_FOUNDRY}) project and
+    is licensed under the [Apache License, Version 2.0]({_LICENSE_APACHE}).
+
+    See the [NOTICE]({_NOTICE}) file for more information.
+
+    What you create with this software is your sole property. All your artwork
+    is free for you to use as you like. That means that the Nope Foundry can be
```
Comment (mbouron) [inline pynopegl-utils/pynopegl_utils/viewer/__init__.py:57]: `Nope Foundry` -> `Nope Viewer`

**Review by mbouron — APPROVED** [review]
Except from my comment, LGTM.

**Pull request merged**